src/styles/utils/reset.css

```diff
@@ -0,0 +1,14 @@
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul {
```

Comment:
It would be better to reset the margin and padding for all elements using the universal selector `*`. This way, you can ensure that all elements have their default margins and paddings removed, not just the ones specified in the code.